docs/guides/postinstall.md

```diff
@@ -184,6 +185,59 @@ S3 suspend has been broken since macOS Sonoma, it has never been fixed, but this
 
 5. If you are facing issues with Wi-Fi on resume, uncomment the lines having `brcmfmac` and `brcmfmac_wcc` in the above file.
 
-!!! note
+> !!! note
     Make sure you have `CONFIG_MODULE_FORCE_UNLOAD=y` in the kernel config.
     To check, run: `zcat /proc/config.gz | grep "CONFIG_MODULE_FORCE_UNLOAD"` on Arch based distros.
+
+### Gentoo/OpenRC
+
+For Gentoo Linux using OpenRC and elogind, create the following script:
+
+For T2 MacBooks, while unloading only the apple-bce module is sufficient for basic suspend functionality, additional module handling may be required depending on your model:
+
+- All T2 models require the apple-bce module handling
+- For models with Touch Bar, a specific module sequence (apple_bce -> hid_appletb_bl -> hid_appletb_kbd -> appletbdrm) is required to properly reinitialize the Touch Bar device after resume
+- If you use tiny-dfr for Touch Bar customization, the tiny-dfr service needs to be stopped before suspend and started after resume
+
+The script below includes all cases with commented sections. Uncomment the relevant sections based on your model and requirements. The loading order of modules is important for proper device initialization after resume.
+
+1. Create and edit this file: `/etc/elogind/system-sleep/apple-bce-handler`
+
+
+```#!/bin/bash
+case $1/$2 in
+  pre/*)
+    # For models with Touch Bar and using tiny-dfr
+    #/etc/init.d/tiny-dfr stop
+    #killall -9 tiny-dfr 2>/dev/null
+    
+    # Required for all T2 models
+    rmmod -f apple_bce
+    
+    # Only for models with Touch Bar
```

Comment:
Write as Uncomment the following for models with touchbar but not using tiny dfr